docs/en/glossary/glossary.asciidoc

```diff
@@ -614,6 +629,14 @@ some other application.
 endif::logstash-terms[]
 ifdef::elasticsearch-terms[]
 
+[[glossary-shrink]] shrink ::
+
+To reduce the amount of shards in an index. See the {es-ref}/current/indices-shrink-index.html[shrink index API].
++
```

Comment:
I'm not familiar with where/if es-ref is defined. Typically use {ref} instead from https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc